web_src/less/_repository.less

```
@@ -717,32 +717,90 @@
                 z-index: -1;
             }
 
-            .timeline-line {
+            .timeline {
                 position: relative;
                 display: block;
-                width: 100%;
-                max-width: 100%;
+                margin-left: 40px;
+                padding-left: 16px;
 
-                &:before {
+                &:before { //ciara
                     display: block;
                     content: "";
                     position: absolute;
                     margin-top: 12px;
                     margin-bottom: 14px;
                     top: 0;
                     bottom: 0;
-                    left: 82px;
+                    left: 30px;
                     width: 2px;
                     background-color: #f3f3f3;
                     z-index: -1;
                 }
             }
 
-            .comment {
-                .avatar {
-                    width: @comment-avatar-width;
+            .timeline-item {
+                padding: 6px 0;
```

```suggestion
                padding: 16px 0;
```
Spacing of the items is too close together, this seems better

---

👍 but there are 2 things that we need to keep in mind, we have to be different from github a bit, and we are not gruping same timeline info so the timeline can be huge, 

btw i'd like to know are any plans to add commits to the timeline?

---

for commit history comments , I have try it in #10680, I will optimize and commit it a few days later.

---

I likethe more compact view - les page scroling - so github has to mouch space inbetween

---

I think this can be too compact and hard to read -- having it a bit larger makes it more accessible for people who would have a difficult time.

Testing on a Github PR with 80 + comments changing the margin from 16 to 6 makes the entire page about 5% shorter which has no real effect on scrolling and I don't think is a noticeable difference for page size since it takes the same amount of time to scroll it either way because a normal scroll covers much more than that small 5%. Making it hard to read to save 5% doesn't feel like a good trade off.

I think that similar activities like commits can be grouped in their own timeline item and not be that far apart but it looks very cluttered currently.

I'm happy to try an in between number but the current one is too small and will be more difficult for some people to read and parse easily.

---

i have adjusted spacing a bit, could you recheck

---

This spacing is ok : )